Dockerfile

```diff
@@ -1,10 +1,6 @@
-# -- trivial container for bgpalerter
-FROM node:12-alpine as build
+FROM ubuntu
```

Comment:
Should you specify the ubuntu version here to lock it in?

Reply:
The docker container to use to build the "BGPAlerter binary is not being used right" as you suggest to build directly on the TC Agent. So you can probably ignore the docker file